SimGeneral/MixingModule/python/aliases_cfi.py

```diff
@@ -78,13 +78,14 @@
 (~phase2_hgcal).toModify(simHGCalUnsuppressedDigis, mix = None)
 
 from Configuration.ProcessModifiers.premix_stage1_cff import premix_stage1
-(premix_stage1 & phase2_hgcal).toModify(simHGCalUnsuppressedDigis,
-    mix = {
-        0 : dict(type = "PHGCSimAccumulator"),
-        1 : dict(type = "PHGCSimAccumulator"),
-        2 : dict(type = "PHGCSimAccumulator"),
-    }
-)
+
+(premix_stage1 & phase2_hgcal).toModify(simHGCalUnsuppressedDigis,                                                                
```

Comment:
please remove all the extra whitespace at the end of these lines

Reply:
@adas1994 this is not addressed yet. at the end of each line (81-87), there is a lot of whitespace that wasn't there before. this should be removed.